### PR TITLE
Fix audio alt files missing bit rate

### LINKED ISSUE #3930: KeyError: "Got KeyError when attempting to get a value for field `bit_rate` on serializer `AudioAltFileSeri...
## Description
The `AudioAltFileSerializer` added with the implementation of Schemathesis tests set all of the alt file fields required.
The newly-added audio provider, CC Mixter, does not have the data for `bit_rate`. This causes any search results that contain CC Mixter results with alt files to return 500 error.

## Sentry Issue
[OPENVERSE-API-D1](https://openverse.sentry.io/issues/5078011270/?referrer=github_integration)

```
KeyError: 'bit_rate'
  File "rest_framework/fields.py", line 446, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "rest_framework/fields.py", line 94, in get_attribute
    instance = instance[attr]

KeyError: "Got KeyError when attempting to get a value for field `bit_rate` on serializer `AudioAltFileSerializer`.\nThe serializer field might be named incorrectly and not match any attribute or key on the `dict` instance.\nOriginal exception text was: 'bit_rate'."
(16 additional frame(s) were not displayed)
...
  File "api/views/media_views.py", line 148, in list
    return self.get_media_results(request, "search", params)
  File "api/views/media_views.py", line 242, in get_media_results
    return self.get_paginated_response(serializer.data)
  File "api/serializers/audio_serializers.py", line 215, in to_representation
    output = super().to_representation(instance)
  File "api/serializers/media_serializers.py", line 584, in to_representation
    output = super().to_representation(*args, **kwargs)
```

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #3930 by @sentry-bot

## Description

This PR marks some audio alt file properties as not required. This fixes the 500 error when serializing CC Mixter items that don't set `bit_rate` on the alt files.

To allow for local tests, this PR adds 10 audio items from CC Mixter to the `sample_audio.csv`

## Testing Instructions

Run the API. Delete the current audio index (I use Elasticvue for this, there is probably a better way to recreate the index) and run `just api/init` to index audio again.
Check that 10 CC Mixter items are rendered correctly: http://localhost:50280/v1/audio/?source=ccmixter
See that this causes 500 without the code changes in this PR, or in prod. 

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [ ] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: api/api/serializers/audio_serializers.py [hunk 1]
```diff
@@ -133,18 +133,19 @@ class Meta:
         help_text="URL of the alternative file.",
     )
     bit_rate = serializers.IntegerField(
-        help_text="Bit rate of the alternative file.",
-        min_value=0,
+        help_text="Bit rate of the alternative file.", min_value=0, required=False
```
Comment (dhruvkb) [inline api/api/serializers/audio_serializers.py:136]: Style nit:

```suggestion
        help_text="Bit rate of the alternative file.", min_value=0, required=False,
```

File: api/api/examples/audio_responses.py [hunk 2]
```diff
@@ -82,7 +82,7 @@
             "display_name": "Freesound",
             "source_url": "https://freesound.org/",
             "logo_url": None,
-            "media_count": 828,
+            "media_count": 827,
```
Comment (dhruvkb) [inline api/api/examples/audio_responses.py:85]: Should we add ccMixter to this stats response with `media_count` = 1?

**Review by dhruvkb — APPROVED** [review]
LGTM! But I have a question and a comment.

File: api/api/examples/audio_responses.py (hunk 2 above)
Comment (obulat) [reply api/api/examples/audio_responses.py:85]: When I added ccMixter, the audio example tests failed. I didn't have much time to look into it. Would it be okay as a separate issue? I'd like to deploy this fix today as this issue is causing a lot of 500s

**Comment by krysal** [comment]
When you say "10 CC Mixter items" do you really mean 1 item? That is what I see in the changes and what I get from the DB. Asking to make sure I'm not missing something.

**Comment by obulat** [comment]
> When you say "10 CC Mixter items" do you really mean 1 item? That is what I see in the changes and what I get from the DB. Asking to make sure I'm not missing something.

Sorry, I forgot to update the description. I first replaced the last 10 items with cc Mixter items. That made the tests fail because, apparently, the integration tests depend on those. I tried finding which ones it was, but then I gave up and only added a single item.

**Comment by krysal** [comment]
Thanks for clarifying it! 👍

**Pull request merged**